請注意，當測試失敗時，worker process 會重新啟動，並且 `afterAll` hook 會在新的 worker 中再次執行。了解更多關於[workers 和失敗](../test-retries.mdx)。